src/net_processing.cpp

```diff
@@ -1397,6 +1397,14 @@ void PeerManagerImpl::FindNextBlocksToDownload(const Peer& peer, unsigned int co
         return;
     }
 
+    // When syncing with AssumeUtxo, if the snapshot is not in the peer's best chain, abort:
```

Comment:
nit: A bit easier to parse IMO, only if you have to retouch. 

```suggestion
    // When we sync with AssumeUtxo and discover the snapshot is not in the peer's best chain, abort:
```

Reply:
done with the latest push.